pkg/controller/jenkins/configuration/base/resources/rbac.go

```diff
@@ -74,3 +53,30 @@ func NewRoleBinding(name, namespace, serviceAccountName string, roleRef v1.RoleR
 		},
 	}
 }
+
+func NewDefaultPolicyRules() []v1.PolicyRule {
+	var rules []v1.PolicyRule
+	Default := []string{createVerb, deleteVerb, getVerb, listVerb, patchVerb, updateVerb, watchVerb}
+	ReadOnly := []string{getVerb, listVerb, watchVerb}
+
+	rules = append(rules,  NewPolicyRule(EmptyApiGroups, "configmaps", Default))
+	rules = append(rules,  NewPolicyRule(EmptyApiGroups, "pods", Default))
+	rules = append(rules,  NewPolicyRule(EmptyApiGroups, "pods/exec", Default))
+	rules = append(rules,  NewPolicyRule(EmptyApiGroups, "pods/portforward", Default))
+	rules = append(rules,  NewPolicyRule(EmptyApiGroups, "pods/log", ReadOnly))
+	rules = append(rules,  NewPolicyRule(EmptyApiGroups, "secrets", ReadOnly))
+	rules = append(rules,  NewPolicyRule(OpenshiftApiGroup, "imagestreams", ReadOnly))
+	rules = append(rules,  NewPolicyRule(BuildApiGroup, "buildconfigs", ReadOnly))
+	rules = append(rules,  NewPolicyRule(BuildApiGroup, "builds", ReadOnly))
```

Comment:
All policies must be reflected here https://github.com/jenkinsci/kubernetes-operator/blob/master/deploy/role.yaml#L6 as well.